src/main/java/hudson/plugins/cobertura/MavenCoberturaPublisher.java

```diff
@@ -221,7 +221,7 @@ private boolean isCoberturaReport(MojoInfo mojo) {
 	 */
 	@Override
 	public Collection<? extends Action> getProjectActions(MavenModule project) {
-		return Collections.singleton(new CoberturaProjectAction(project));
+		return Collections.singleton(new CoberturaProjectAction(project.getLastBuild()));
```

Comment:
Probably needs a null check, and possibly a `result` check too (like `getLastSuccessfulBuild`).